docs/user/alerting/action-types.asciidoc

```diff
@@ -27,6 +27,10 @@ a| <<slack-action-type, Slack>>
 
 | Send a message to a Slack channel or user.
 
+a| <<servicenow-action-type, ServiceNow>>
+
+| Create new ServiceNow incident.
```

Comment:
The copy here should match what is displayed in the connector flyout. You can check with @gchaps if we didn't get copy for a description of ServiceNow.

<img width="267" alt="Screen Shot 2020-07-14 at 10 44 58 AM" src="https://user-images.githubusercontent.com/3694571/87439688-15007b80-c5bf-11ea-8bd7-235115a71ed5.png">